src/plugins/data/server/index_patterns/routes.ts

```diff
@@ -69,15 +69,25 @@ export function registerRoutes(
       },
     },
     async (context, request, response) => {
-      const { asCurrentUser } = context.core.elasticsearch.client;
-      const indexPatterns = new IndexPatternsFetcher(asCurrentUser);
+      const savedObjectsClient = context.core.savedObjects.client;
```

Comment:
I wonder if we can keep `_fields_for_wildcard` as only a call to `IndexPatternsFetcher`